src/widgets/Notebook.cpp

```diff
@@ -1154,6 +1156,28 @@ void SplitNotebook::addCustomButtons()
     auto updateBtn = this->addCustomButton();
 
     initUpdateButton(*updateBtn, this->signalHolder_);
+
+    // streamer mode
+    auto *streamerModeIcon = this->addCustomButton();
+    if (getTheme()->isLightTheme())
+    {
+        streamerModeIcon->setPixmap(
+            getResources().buttons.streamerModeEnabledLight);
+    }
+    else
+    {
+        streamerModeIcon->setPixmap(
+            getResources().buttons.streamerModeEnabledDark);
+    }
+    streamerModeIcon->setVisible(isInStreamerMode());
+    this->signalHolder_.managedConnect(
+        getApp()->streamerModeChanged, [streamerModeIcon]() {
+            streamerModeIcon->setVisible(isInStreamerMode());
+        });
+    QObject::connect(streamerModeIcon, &NotebookButton::leftClicked, [this] {
+        getApp()->windows->showSettingsDialog(
+            this, SettingsDialogPreference::StreamerMode);
+    });
```

Comment:
could you move this to right below the `addCustomButton` thing? (below line 1161)
that way, line 1162 and onwards is exactly the same for windows